indexd/index/blueprint.py

```diff
@@ -474,7 +474,10 @@ def health_check():
     '''
     Health Check.
     '''
+
     blueprint.index_driver.health_check()
+    flask.current_app.config['INDEX']['driver'].health_check()
```

Comment:
is this supposed to invoke the new alias healthcheck? should that be 
```suggestion
    flask.current_app.config['ALIAS']['driver'].health_check()
```
or am i misreading 🤔